README.md

```diff
@@ -89,6 +89,11 @@ loading(PATH,[{call:true}]).into(TARGET, FIELD, [filters])
   decide whether override the property that is already exist,
   default: false
 
+- **ignore** type:[`string function`]
+  ignore file. If ignore is string, ignore role will be glob string `!ignore`,
```

Comment:
I think string is enough, glob support many patterns. and you can support array that can get multi glob.